src/framework-configuration.js

```diff
@@ -146,8 +176,7 @@ export class FrameworkConfiguration {
    * @return Returns the current FrameworkConfiguration instance.
   */
   feature(plugin: string, config?: any): FrameworkConfiguration {
-    plugin = plugin.endsWith('.js') || plugin.endsWith('.ts') ? plugin.substring(0, plugin.length - 3) : plugin;
-    return this.plugin({ moduleId: plugin + '/index', resourcesRelativeTo: plugin, config: config || {} });
+    return this.plugin({ moduleId: plugin + '/index', resourcesRelativeTo: [plugin, ''], config: config || {} });
```

Comment:
One problem I see here though. As we are always appending `/index`, the "final" moduleId may end up looking like `some-feature.ts/index` which is surely an invalid modueId. @EisenbergEffect